ts/lib/auth/jwtclient.ts

```diff
@@ -74,7 +75,8 @@ export default class JWT extends Auth2Client {
    * @param {string} opt_uri the URI being authorized.
    * @param {function} metadataCb
    */
-  public getRequestMetadata(opt_uri: string, metadataCb) {
+  public getRequestMetadata(
+      opt_uri: string, metadataCb: (err: Error, result?: any) => void) {
```

Comment:
I don't think `uri?` would be appropriate, since the metadataCb is actually required as written.  The `?` only really works on trailing properties, right?